content/en/docs/Kyverno CLI/_index.md

```diff
@@ -224,3 +224,159 @@ policies:
+| 4.   | policy.yaml   | -r resourceName  | true      | -n=namespace   | apply policy from file to the resource in cluster in mentioned namespace                 |
+| 5.   | policy.yaml   |                  | true      | -n=namespace   | apply policy from file to all the resources in cluster in mentioned namespace            |
+
+
+Example:
+
+Consider the following policy and resources:
+
+policy.yaml
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: require-pod-requests-limits
+  annotations:
+    policies.kyverno.io/category: Workload Management
+    policies.kyverno.io/description: >-
+      As application workloads share cluster resources, it is important to limit resources
+      requested and consumed by each pod. It is recommended to require 'resources.requests'
+      and 'resources.limits' per pod. If a namespace level request or limit is specified,
+      defaults are automatically applied to each pod based on the 'LimitRange' configuration.
+spec:
+  validationFailureAction: audit
+  rules:
+  - name: validate-resources
+    match:
+      resources:
+        kinds:
+        - Pod
+    validate:
+      message: "CPU and memory resource requests and limits are required"
+      pattern:
+        spec:
+          containers:
+          - resources:
+              requests:
+                memory: "?*"
+                cpu: "?*"
+              limits:
+                memory: "?*"
+```
+
+resource1.yaml
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx1
+  labels:
+    env: test
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    imagePullPolicy: IfNotPresent
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+```
+
+resource2.yaml
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx2
+  labels:
+    env: test
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    imagePullPolicy: IfNotPresent
+```
+
+Case 1: Apply policy manifest on resource manifest
+
+```
+kyverno apply policy.yaml -r resource1.yaml -r resource2.yaml --policy_report
+```
+
+Case 2: Apply policy manifest on cluster resource 
+
+create above resource manifest in cluster.
+```
+kyverno apply policy.yaml -r nginx1 -r nginx2 --cluster --policy_report
+```
+
+Case 3: Apply policy manifest on all resource avaliable in cluster
+```
+kyverno apply policy.yaml --cluster --policy_report
+```
+This will validate all the pods avaliable in cluster.
+
+Case 4: Apply policy manifest on resource avaliable in cluster under a specific namspace
+```
+kyverno apply policy.yaml -r nginx1 -r nginx2 --cluster --policy_report -n default
+```
+
+Case 5: Apply policy manifest on all resource avaliable in cluster under a specific namespace
+```
+kyverno apply policy.yaml --cluster --policy_report -n default
+```
+This will validate all the pods avaliable in cluster avaliable under default namespace.
```

Comment:
"Given the contents of `policy.yaml` shown earlier, this will produce a report validating all Pods within the `default` Namespace."